### PR TITLE
[fuchsia] Detect and enable KVM.

### PR DESCRIPTION
**Pull request by flowerhack**
Detecting KVM is tricky, so let's use an environment variable (to
generally be set by the docker image) to determine whether it's safe to
turn KVM on for Fuchsia VMs.

### CODE REVIEW

**Comment by flowerhack** [comment]
@oliverchang

File: src/python/platforms/fuchsia/device.py [hunk 1]
```diff
@@ -113,6 +113,12 @@ def qemu_setup():
   ]
   # yapf: enable
 
+  # Detecing KVM is tricky, so let's use an environment variable, set from the
+  # docker image, to determine whether to turn it on or not.
+  kvm = environment.get_value('FUCHSIA_USE_KVM')
```
Comment (oliverchang) [inline src/python/platforms/fuchsia/device.py]: nit: no need for the variable. just use get_value in the if.

**Pull request merged**